ingestors/support/convert.py

```diff
@@ -1,21 +1,25 @@
-import math
 import logging
-import requests
-from requests import RequestException, HTTPError
-from servicelayer.util import backoff
+import os
+import pathlib
+import subprocess
+from tempfile import gettempdir
+
 from followthemoney.helpers import entity_filename
 
-from ingestors.settings import CONVERT_URL, CONVERT_TIMEOUT, CONVERT_RETRIES
 from ingestors.support.cache import CacheSupport
 from ingestors.support.temp import TempFileSupport
-from ingestors.util import explicit_resolve
 from ingestors.exc import ProcessingException
 
 log = logging.getLogger(__name__)
 
+CONVERT_DIR = os.path.join(gettempdir(), "convert")
+OUT_DIR = os.path.join(CONVERT_DIR, "out")
+TIMEOUT = 3600
```

Comment:
Is this in milliseconds or seconds?

Reply:
seconds, will add a comment 👍

Reply:
and perhaps lower it to something reasonable? 600?

Reply:
Hmmm 10 minutes sounds reasonable for reasonably sized PDF documents. But what happens when the PDF is 1GB+ in size. Let's keep the TIMEOUT painfully big until we can review how quickly we can successfully process a thicc pdf.